ext_localconf.php

```diff
@@ -47,3 +47,27 @@
     \B13\Newspage\Filter\CategoriesFilter::class,
     'LLL:EXT:newspage/Resources/Private/Language/locallang_be.xlf:filter.categories'
 );
+
+$icons = [
+    'apps-pagetree-newspage-page' => 'apps-pagetree-newspage-page',
+    'apps-pagetree-newspage-page-hideinmenu' => 'apps-pagetree-newspage-page-hideinmenu',
+    'apps-pagetree-newspage-article' => 'apps-pagetree-newspage-article',
+    'apps-pagetree-newspage-article-hideinmenu' => 'apps-pagetree-newspage-article-hideinmenu',
+    'apps-pagetree-newspage-category' => 'apps-pagetree-newspage-category',
+    'apps-pagetree-newspage-category-hideinmenu' => 'apps-pagetree-newspage-category-hideinmenu',
+    'apps-pagetree-newspage-overview' => 'apps-pagetree-newspage-overview',
+    'apps-pagetree-newspage-overview-hideinmenu' => 'apps-pagetree-newspage-overview-hideinmenu',
+    'apps-pagetree-newspage-tag' => 'apps-pagetree-newspage-tag',
+    'apps-pagetree-newspage-tag-hideinmenu' => 'apps-pagetree-newspage-tag-hideinmenu',
+    'mimetypes-newspage' => 'mimetypes-newspage'
+];
+$iconRegistry = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance(\TYPO3\CMS\Core\Imaging\IconRegistry::class);
```

Comment:
When declaring variables in any of the configuration files, I prefer them to be enclosed in a function block.
This makes sure that they are not kept in memory and are "available" in other PHP files.